connectivity/conn_http.h

```diff
@@ -23,31 +23,123 @@ extern "C" {
 #include "mbedtls/net.h"
 #include "mbedtls/ssl.h"
 
-#define HTTP_OK 200
-
+/**< Global pointer points to http resource body for passing response body to http_parser.
+ * This will be removed or change to static pointer as soon as possible. Don't use this
```

Comment:
This variable will be removed or changed

Reply:
I care about the grammar.